jupyterhub_config.py

```diff
@@ -186,11 +187,9 @@ def finalize_user_content(spawner):
 # c.DockerSpawner.extra_create_kwargs.update({ 'command': spawn_cmd })
 
 # Connect containers to this Docker network
-network_name = os.environ['DOCKER_NETWORK_NAME']
+network_name = 'bridge'
 c.DockerSpawner.use_internal_ip = True
-c.DockerSpawner.network_name = network_name
-# Pass the network name as argument to spawned containers
-c.DockerSpawner.extra_host_config = { 'network_mode': network_name }
+c.DockerSpawner.network_name = os.environ['DOCKER_NETWORK_NAME'] 
```

Comment:
Should this use `network_name`, as in (untested):

```suggestion
c.DockerSpawner.network_name = os.environ.get('DOCKER_NETWORK_NAME', network_name)
```